docker/Dockerfile.dev

```diff
@@ -225,5 +225,30 @@ setopt HIST_FIND_NO_DUPS
 setopt INC_APPEND_HISTORY
 EOF
 
-# Set workspace directory
+# Create non-root user with specified UID and GID
```

Comment:
Could you consider using this method? https://github.com/sgl-project/sglang/blob/main/docs/developer/development_guide_using_docker.md#setup-docker-container

Reply:
```bash
docker pull lmsysorg/sglang:dev
docker run -itd --shm-size 32g --gpus all -v /opt/dlami/nvme/.cache:/root/.cache --ipc=host --name sglang_zhyncs lmsysorg/sglang:dev /bin/zsh
docker exec -it sglang_zhyncs /bin/zsh
```

Reply:
After that, you can use VSCode to attach to the container.

Reply:
Thanks! I am using [dev container](https://code.visualstudio.com/docs/devcontainers/containers) extension. The permission issue seems to happen with both method. Maybe it is something related to my VM setup. I will use this as a temp solution but we don't have to merge the code